src/store/traction/saphyr/tubes/mutations.js

```diff
@@ -1,6 +1,9 @@
 const mutations = {
   setTubes (state, tubes) {
     state.tractionTubes = tubes
+  },
```

Comment:
could add small tests for these, don't think any exist at the moment

Reply:
Again I think we are missing tests for a few mutations.